lib/LedgerBlockStorage.js

```diff
@@ -140,22 +140,21 @@ class LedgerBlockStorage {
    */
   async get({blockId, consensus = true}) {
     // find an existing block with consensus
+    // audit:storage-mongodb/646f54f5-ac4e-40d3-abe7-0af35449c3d1.md
     const query = {
       id: database.hash(blockId),
-      'meta.deleted': {
-        $exists: false
-      },
       'meta.consensus': consensus
     };
-    const record = await this.collection.findOne(query);
-    if(!record) {
+    const projection = {_id: 0};
+    const {block, meta} = await this.collection.findOne(query, projection);
```

Comment:
Can't the result of `findOne` be `null`? If so, it seems like this would throw based on a simple test of:

`const {a, b} = null;`

Same applies to other code below. Maybe need to do:

```
const {block, meta} = (await this.collection.findOne(query, projection)) || {});
```

or 

```
const record = await this.collection.findOne(query, projection));
if(!record) {
  // ...
}
const {block, meta} = record;
```

Reply:
doh! fixed.